omnibus/config/software/datadog-process-agent.rb

```diff
@@ -13,19 +13,27 @@
 end
 default_version process_agent_version
 
+url = "https://s3.amazonaws.com/datad0g-process-agent/"
+
 build do
   if windows?
     binary = "process-agent-windows-#{version}.exe"
     target_binary = "process-agent.exe"
-    url = "https://s3.amazonaws.com/datad0g-process-agent/#{binary}"
-    curl_cmd = "powershell -Command wget -OutFile #{binary} #{url}"
+    curl_cmd = "powershell -Command wget -OutFile #{binary} #{url}#{binary}"
     command curl_cmd
     command "mv #{binary} #{install_dir}/bin/agent/#{target_binary}"
   else
     binary = "process-agent-amd64-#{version}"
     target_binary = "process-agent"
-    url = "https://s3.amazonaws.com/datad0g-process-agent/#{binary}"
-    curl_cmd = "curl -f #{url} -o #{binary}"
+    curl_cmd = "curl -f #{url}#{binary} -o #{binary}"
```

Comment:
just noticed that this software definition doesn't ship the licenses of the software it adds, which is a Bad Thing :)

Could you use the `ship_license` DSL method to add the licenses of the process agent and network tracer? (you'll find examples in the other software definitions in this directory). You can use it multiple times to add multiple licenses (although if the filenames are the same they might overwrite one another)

Once that's added, the licenses should be located at `/opt/datadog-agent/datadog-process-agent/` in the resulting linux packages

Reply:
Licenses should be good to go!